cmake/modules/CoverageReport.cmake

```diff
@@ -61,4 +70,4 @@ add_custom_target(generate-coverage-report
                           ${REPORT_DIR} ${coverage_binaries}
                           --unified-report ${restrict_flags}
                   WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-                  DEPENDS check-llvm) # Run tests
+                  ${COV_DEPENDS}) # Run tests
```

Comment:
This looks like it changes the default behaviour, since `COV_DEPENDS` is empty if `LLVM_CODE_COVERAGE_TEST_TARGETS` isn't set.